js/index.js

```diff
@@ -85,3 +85,63 @@ messageForm.addEventListener('submit',function(event){
     };
 
  });
+
+//  let githubRequest = new XMLHttpRequest();
```

Comment:
Feel free to delete this if you would like. I don't think we are going to need it anymore